products/appengine/api.yaml

```diff
@@ -21,6 +21,119 @@ versions:
+    base_url: 'apps/{{project}}/domainMappings'
+    self_link: 'apps/{{project}}/domainMappings/{{domain_name}}'
+    update_verb: :PATCH
+    update_mask: true
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/appengine/docs/standard/python/mapping-custom-domains'
+      api: 'https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.domainMappings'
+    async: !ruby/object:Api::Async
+      operation: !ruby/object:Api::Async::Operation
+        path: 'name'
+        base_url: '{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::Async::Result
+        path: 'response'
+        resource_inside_response: true
+      status: !ruby/object:Api::Async::Status
+        path: 'done'
+        complete: True
+        allowed:
+          - True
+          - False
+      error: !ruby/object:Api::Async::Error
+        path: 'error'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::Enum
+        name: 'overrideStrategy'
+        description: |
+          Whether the domain creation should override any existing mappings for this domain.
+          By default, overrides are rejected.
+        url_param_only: true
+        values:
+          - :UNSPECIFIED_DOMAIN_OVERRIDE_STRATEGY
+          - :STRICT
+          - :OVERRIDE
+        default_value: :STRICT
+      - !ruby/object:Api::Type::String
+        name: 'id'
+        description: |
+          Relative name of the domain serving the application. Example: example.com.
+        required: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          Full path to the DomainMapping resource in the API. Example: apps/myapp/domainMapping/example.com.
+        output: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'sslSettings'
+        description: |
+          SSL configuration for this domain. If unconfigured, this domain will not serve with SSL.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'certificateId'
+            description: |
+              ID of the AuthorizedCertificate resource configuring SSL for the application. Clearing this field will
+              remove SSL support.
+              By default, a managed certificate is automatically created for every domain mapping. To omit SSL support
+              or to configure SSL manually, specify `SslManagementType.MANUAL` on a `CREATE` or `UPDATE` request. You must be
+              authorized to administer the `AuthorizedCertificate` resource to manually map it to a DomainMapping resource.
+              Example: 12345.
+          - !ruby/object:Api::Type::Enum
+            name: 'sslManagementType'
+            description: |
+              SSL management type for this domain. If `AUTOMATIC`, a managed certificate is automatically provisioned.
```

Comment:
I believe so.  That was one of the things I was trying to test, because as I was testing this I never actually saw `certificateId` be populated, but it seems like it should be.  But I was getting that provisioning error, so I assumed that's why.